src/apps/common/submodules/callerId/callerId.js

```diff
@@ -51,36 +73,75 @@ define(function(require) {
 			popup_html.find('.save').on('click', function(ev) {
 				ev.preventDefault();
 
-				if (monster.ui.valid(form)) {
-					var cnamFormData = monster.ui.getFormData('cnam');
+				if (!monster.ui.valid(form)) {
+					return;
+				}
 
-					_.extend(dataNumber, { cnam: cnamFormData });
+				_.extend(dataNumber, {
```

Comment:
`lodash#extend` is legacy code from the migration to lodash (from underscore).

Although it [achieves the intended goal](https://lodash.com/docs/#assignIn) in this case, let's try to move away from it locally and use the more appropriate method (only assign own props vs own + inherited props).
```suggestion
				_.assign(dataNumber, {
```